di-ipv-core-stub/src/main/java/uk/gov/di/ipv/stub/core/CoreStub.java

```diff
@@ -58,11 +59,25 @@ private ExceptionHandler exceptionHandler() {
 
     private RSAKey getSigningKeystore() throws Exception {
         KeyStore keystore = KeyStore.getInstance("pkcs12");
-        try (ByteArrayInputStream inputStream =
-                new ByteArrayInputStream(
-                        Base64.getDecoder().decode(CoreStubConfig.CORE_STUB_KEYSTORE_BASE64))) {
-            keystore.load(inputStream, CoreStubConfig.CORE_STUB_KEYSTORE_PASSWORD.toCharArray());
+
+        if (CoreStubConfig.CORE_STUB_KEYSTORE_FILE != null) {
+            try (FileInputStream inputStream =
+                    new FileInputStream(CoreStubConfig.CORE_STUB_KEYSTORE_FILE)) {
+                keystore.load(
+                        inputStream, CoreStubConfig.CORE_STUB_KEYSTORE_PASSWORD.toCharArray());
+            }
+        } else if (CoreStubConfig.CORE_STUB_KEYSTORE_BASE64 != null) {
```

Comment:
Heads up but looking at the current deployment in PaaS, this has already been set